modules/end-user-guide/partials/ref_devfile-reference.adoc

```diff
@@ -7,6 +7,56 @@
 
 This section contains devfile reference and instructions on how to use the various elements that devfiles consist of.
 
+[id="adding-schema-version-to-a-devfile_{context}"]
+== Adding schema version to a devfile
+
+.Procedure
+
+* Define the `schemaVersion` attribute in the devfile:
+
+.Adding schema version to a devfile
+====
+[source,yaml]
+----
+schemaVersion: 2.0.0
```

Comment:
Do we target devfile 1 or 2? This is of devfile 2. It has impact on when we can merge.

Reply:
Removed 7.24.x as milestone, if we want to document devfile 2

Reply:
Suggest keeping this one on hold until the new version rolls out with the full support of devfile 2. I believe it's 2.27, no?

Reply:
You would probably want to use `schemaVersion: 1.0.0` as an example. Current documentation is about the v1.0.0 spec (the syntax in v2 is sensibly different and not backward compatible).

Anyway since Che 7.27.1 we have v2 support but that's disable by default (the admin that deploys Che has to explicitly enable it to allow developers to use v2 of the spec).

Reply:
:+1: to use `schemaVersion: 1.0.0` for now

Reply:
done